lti_tool/utils.py

```diff
@@ -0,0 +1,57 @@
+from django.conf import settings
+from datetime import datetime
+from pylti1p3.grade import Grade
+from pylti1p3.contrib.django import DjangoMessageLaunch, DjangoCacheDataStorage, DjangoDbToolConf
+from pylti1p3.lineitem import LineItem
+from django.contrib.auth.models import User
+from userprofile.models import UserProfile
+from course.models import Enrollment
+
+def get_launch_url(request):
```

Comment:
Recently we have applied type hints at method definitions. Could do so also in these new lti_tool modules, e.g.
`def get_launch_url(request: HttpRequest) -> str:`

Reply:
Now that we are running out time, I think we have to skip the static type hinting.